src/commands/up/app_source.rs

```diff
@@ -83,7 +83,7 @@ pub enum ResolvedAppSource {
 }
 
 impl ResolvedAppSource {
-    pub fn trigger_type(&self) -> anyhow::Result<&str> {
+    pub fn trigger_type(&self) -> anyhow::Result<Vec<&str>> {
```

Comment:
I know I am getting into the weeds at this point, but this will need to be renamed `trigger_types`